src/ShoppingCart.java

```diff
@@ -9,31 +9,60 @@ public static void main(String[] args) {
         String input = "";
         System.out.println("Welcome to Pierre's store. Please choose what items you'd like from the list below. Some items may not be in stock.");
 
-        String list[] = {"1.Nintendo Switch - $299.99 ",
-                "2.Xbox Series X - $499.99 ",
-                "3.PS5 - $499.99 ",
-                "4.Samsung Q60A TV - $440.00 ",
-                "5.Apple Watch - #399.99 ",
-                "6.GTX3070 - $499.99 "
-        };
-
-
+        List<Item> itemShelf = new ArrayList<>();
+        List<Item> cart = new ArrayList<>();
         //example item
         Item nintendoSwitch = new Item(1, "Nintendo Switch", 299.99);
+        Item xboxSeriesX = new Item(2, "Xbox Series X", 499.99);
+        Item ps5 = new Item(3, "PS5", 499.99);
+        Item samsungQ60atv = new Item(4, "Samsung Q60A", 440.00);
+        Item appleWatch = new Item(5, "Apple Watch", 399.99);
+        Item gtx3070 = new Item(6, "GTX3070", 499.99);
 
-        boolean addFlag = true;
+        itemShelf.add(nintendoSwitch);
+        itemShelf.add(xboxSeriesX);
+        itemShelf.add(ps5);
+        itemShelf.add(samsungQ60atv);
+        itemShelf.add(appleWatch);
+        itemShelf.add(gtx3070);
 
-        while(addFlag == true){
-            //add logic
+        for (int i = 0; i < itemShelf.size(); i++) {
+            System.out.println(itemShelf.get(i).toString());}
```

Comment:
Instead of just printing the item details, could it be possible to displaying this information nicely?